### PR TITLE
ADEN-2289 Hop in interstitial

### PR DESCRIPTION
**Pull request by fraszczakszymon**
Don't merge it please

### CODE REVIEW

File: resources/wikia/ui_components/modal/js/modal.js [hunk 1]
```diff
@@ -173,7 +174,7 @@ define('wikia.ui.modal', [
 		this.$element = $(jQuerySelector);
 		this.$content = this.$element.children('section');
 		this.$close = this.$element.find('.' + CLOSE_CLASS);
-		this.$blackout = $('#' + blackoutId);
+		this.$blackout = $('#' + blackoutId).addClass(BLACKOUT_HIDDEN_CLASS);
```
Comment (nandy-andy) [inline resources/wikia/ui_components/modal/js/modal.js]: What do you think about having this class added by default in the template and then just removing it with jQuery?

File: extensions/wikia/AdEngine/js/template/modal.js [hunk 2]
```diff
@@ -50,9 +92,25 @@ define('ext.wikia.adEngine.template.modal', [
...
+				size: {
+					width: params.width,
+					height: params.height
+				}
+			},
+			modalHandler,
 			skin = adContext.getContext().targeting.skin,
 			lightboxParams = lightBoxExpansionModel[skin];
 
+		if (skin === 'mercury') {
```
Comment (nandy-andy) [inline extensions/wikia/AdEngine/js/template/modal.js]: What do you think about extracting this whole skin-dependent-handler code to another module?

File: resources/wikia/ui_components/modal/js/modal.js (hunk 1 above)
Comment (fraszczakszymon) [inline resources/wikia/ui_components/modal/js/modal.js]: :+1:

File: extensions/wikia/AdEngine/js/template/modal.js (hunk 2 above)
Comment (fraszczakszymon) [inline extensions/wikia/AdEngine/js/template/modal.js]: :+1: great idea, I've created modalHandlerFactory and modules for oasis and mercury

**Comment by nandy-andy** [comment]
LGTM :+1:

**Pull request merged**